### PR TITLE
Add support for block strings (triple quotes)

### PR DESCRIPTION
**Pull request by fermads**
As per [GraphQL RFC](https://facebook.github.io/graphql/draft/#sec-String-Value) and its [implementation](https://github.com/facebook/graphql/pull/327), block strings are part of the spec for a while as a way to add multi-line documentation to GraphQL types. This PR adds syntax highlight support for triple quotes (""") block strings.

### CODE REVIEW

File: GraphQL.sublime-syntax
Comment (fermads) [inline GraphQL.sublime-syntax:27]: Needed for normal strings (single double-quotes) to not match multi-line strings (triple double-quotes)

**Comment by Conjuror** [comment]
This is helpful. Please consider to merge it :)

**Comment by fermads** [comment]
I don't have merge access 😢 Waiting for the repo owner to approve.

**Review by dncrews — APPROVED** [review]

**Comment by dncrews** [comment]
Sorry mate. I don't know how I never saw this originally.

**Pull request merged**
